checkstyle.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
```

Comment:
This checkstyle is not the google recommended one, could you please add all the rules and then use suppressions to skip some of them